amethyst_rendy/src/debug_drawing.rs

```diff
@@ -96,6 +96,207 @@ impl DebugLinesComponent {
         self.lines.push(vertex);
     }
 
+    /// Adds multiple lines that form a square to be rendered by giving a z coordinate, a min and a max position.
+    pub fn add_square(&mut self, min: Point2<f32>, max: Point2<f32>, z: f32, color: Srgba) {
+        self.add_line(
+            [min[0], min[1], z].into(),
+            [max[0], min[1], z].into(),
+            color,
+        );
+        self.add_line(
+            [min[0], min[1], z].into(),
+            [min[0], max[1], z].into(),
+            color,
+        );
+        self.add_line(
+            [max[0], min[1], z].into(),
+            [max[0], max[1], z].into(),
+            color,
+        );
+        self.add_line(
+            [min[0], max[1], z].into(),
+            [max[0], max[1], z].into(),
+            color,
+        );
+    }
+
+    /// Adds multiple lines that form a rotated square to be rendered by giving a z coordinate, a rotation, a min and a max position.
+    pub fn add_rotated_square(
+        &mut self,
+        min: Point2<f32>,
+        max: Point2<f32>,
+        z: f32,
+        angles: Vector3<f32>,
```

Comment:
rotation should be passed as a `UnitQuaternion`. You can construct that from euler angles on your own if you need to. It's easy to get quaternions out of transforms btw.